include/vigra/blockwise_labeling.hxx

```diff
@@ -88,7 +92,8 @@ blockwiseLabeling(DataBlocksIterator data_blocks_begin, DataBlocksIterator data_
                   LabelBlocksIterator label_blocks_begin, LabelBlocksIterator label_blocks_end,
                   NeighborhoodType neighborhood, Equal equal,
                   const Value* background_value,
-                  Mapping& mapping)
+                  Mapping& mapping,
+                  const int nThreads = ParallelOptions::Auto)
```

Comment:
One could just added, so far I added it just to the HL function of watershed.
But should be easy as pie